lib/swagger/blocks/nodes/root_node.rb

```diff
@@ -54,6 +54,13 @@ def server(inline_keys = nil, &block)
           self.data[:servers] << Swagger::Blocks::Nodes::ServerNode.call(version: version, inline_keys: inline_keys, &block)
         end
 
+        def x(name, inline_keys = nil, &block)
```

Comment:
What do you think about changing the method name from `x` to `extension`? I think that it would be more explicit and provide better self documentation.

Reply:
Hey, thanks for your swift review! Indeed, `extension` would be more clear, particularly if you are not deep into the syntax yet. Let me change that.

Reply:
sorry for the commit mess 😳, I started ruby with 2.2.3 🙊